### ISSUE TITLE
Issue with short-hand values and values of 0

### ISSUE THREAD

**Issue by bgriffith** [sasstools/sass-lint#263]
The `shorthand-values` rule doesn't work with values of zero.

Best way to replicate this is to add the following to the tests:

```scss
.value-of-zero {
  border-radius: 0;
  border-width: 0;
  margin: 0;
  padding: 0;
}
```

It currently returns false warnings for all of these.

<img width="776" alt="screen shot 2015-10-08 at 16 51 36" src="https://cloud.githubusercontent.com/assets/2854338/10371788/e729ea82-6ddc-11e5-9904-e47fdcf5f579.png">

Sass-lint: dev@latest

**Comment by DanPurdy** [comment]
Very odd, will look at this now
